backend/core/src/listings/dto/listing.dto.ts

```diff
@@ -708,6 +711,24 @@ export class ListingUpdateDto extends OmitType(ListingDto, [
   result?: AssetUpdateDto
 }
 
+// The names of supported filters on /listings
+export enum ListingFilterKeys {
+  status = "status",
+  name = "name",
+  neighborhood = "neighborhood",
+}
+
+// This is a hack: We wrap the enum in a class, so it can be included with
+// @ApiExtraModels and picked by the backend-swagger.ts codegen.
+export class ListingsFilterKeysTransporter {
+  @Expose()
+  @ApiProperty({
+    enum: Object.keys(ListingFilterKeys),
+    required: false,
+  })
+  listingFilterKey?: ListingFilterKeys
+}
+
```

Comment:
@pbn4 in order to get the ListingsFilterKeys to be picked up by the swagger codegen, I needed to wrap them in a model. which is a bit of a hack.

A better alternative would probably be to put this enum in a util file that both the backend and frontend import. Is there a place that would be?

Reply:
You can always put it in somewhere in `types/src` and export it through `types/src/index.ts` or do the hacky way the same way as in `backend/core/src/applications/application-preference-api-extra-models.ts`. BTW between latest release and now I think I used IDE to refactor some of the models and I didn't notice that `backend/core/index.ts` exports nothing. Could you add `export * from ./types` to your PR, please? Without it the next package release will contain no types. cc: @seanmalbert

Reply:
Where do I need to add `export * from ./types`? in `types/src/index.ts`?

Reply:
I believe he means to `backend/core/index.ts`

Reply:
Done.